soundlib/Load_stm.cpp

```diff
@@ -161,6 +160,8 @@ bool CSoundFile::ReadSTM(FileReader &file, ModLoadingFlags loadFlags)
 
 	m_nDefaultTempo = ConvertST2Tempo(initTempo);
 	m_nDefaultSpeed = initTempo >> 4;
+
+	m_nDefaultGlobalVolume = 0x40 * 4u;
 	if(fileHeader.verMinor > 10)
```

Comment:
Code would be easier to follow if this was added to the else branch of the following if.
However I am not sure if we want to do this at all, @sagamusix will have an opinion on that.

Reply:
The default value is identical with OpenMPT's default value, which means there is no need to set it explicitely.